### DISCUSSION TITLE
Batch Renaming Based on Position

### DISCUSSION THREAD

**Discussion by Wikinger85** [Stirling-Tools/Stirling-PDF#432] [Q&A] [answered]
Dear Community,

is there any chance to define which text (specified coordinates/position) of the document is used for renaming?

At the moment it took the text in 5th or 6th line but the important information is for me in line 1-2.

Thank you for any help/advice.

Cheers

**Comment by Frooodle** [comment]
I am surprised the auto renamer took the lower line it tries to go off front size? Can I know the kinda format your pdf is in for it to choose the wrong line?

**Reply by Wikinger85** [reply]
As per details its PDF 1.4. Is that the Information you need?

I can mark and copy the first 3 lines and use it as text so it seems that its already "OCR-ready".

**Reply by Frooodle** [reply]
Sorry I mean the true document formatting

Like what text is on what line with what font size etc

Would you be able to screenshot the first part of the page

**Comment by Wikinger85** [comment]
![grafik](https://github.com/Frooodle/Stirling-PDF/assets/118928174/f6b44353-f6c2-4454-b455-41bc7feb7d41)

"LH A340 Outbound Exception Multi Leg Flight (3 Legs) with In-
Between-Snack & Exchange at 1. Destination - A340-313" 

This is the actual part I want to use as filename.

**Reply by Frooodle** [reply, marked as answer]
Ahhh I see 

I will add a custom filter to choose what lines to scan
Right now it scans top 5 lines for largest font

**Reply by Wikinger85** [reply]
Wow great. Thank you for your effort!

**Reply by Wikinger85** [reply]
One last question...will it be then possible to set a custom area where the name is based of? e.g. Specified Coodinates in a certain line with a fix amount of signs? I know its a luxury request. :) Hopefully you know what I try to explain. :)
